regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
```

Comment:
Yes, you're right, it is, and the particular definition this was an attempt to match is handled by `INCLUDE_DEFINITIONS_IN`. Reverting this change.